_rules/aria-hidden-no-focusable-content-6cfa84.md

```diff
@@ -169,6 +184,27 @@ This `summary` element is [focusable][].
 </details>
 ```
 
+#### Failed Example 7
+
+This `a` element is [focusable][] because it fails to move focus when it receives focus. This is in contrast to a focus sentinel that 
+immediately jumps focus to a valid location. Focus sentinels are typically used before and after modal dialogs in order to contain 
```

Comment:
I don't think "dialogs" is a word. If you want to use it, add it to `__tests__/spelling-ignore.yml` so the test doesn't fail it.

Reply:
Changed to "after a modal dialog"